rootfs/requirements.txt

```diff
@@ -10,7 +10,8 @@ gunicorn==19.4.1
 psycopg2==2.6.1
 python-etcd==0.3.2
 PyYAML==3.11
-requests==2.8.1
-simpleflock==0.0.2
+requests==2.9.1
+#simpleflock==0.0.2
+git+https://github.com/helgi/python-simpleflock.git@python3#egg=simpleflock
```

Comment:
The current Kubernetes 1.1.3 has Docker 1.8.2, at least when I deploy it according to the official vagrant instructions. And I don't think the "concurrent pull" bug was truly fixed until Docker 1.9, so Deis v2 will have to keep working around this.

(You can probably tell just by watching your k8s pods start up that Kubernetes itself had to implement the same workaround: all `docker pull` operations are serialized on a node.)